frontend/package.json

```diff
@@ -33,20 +33,26 @@
     "@testing-library/jest-dom": "4.2.4",
     "@testing-library/react": "9.3.3",
     "@testing-library/user-event": "7.1.2",
+    "@types/enzyme": "3.10.5",
+    "@types/enzyme-adapter-react-16": "1.0.6",
     "@types/jest": "24.0.23",
-    "@types/react-test-renderer": "16.9.2",
-    "@types/redux-mock-store": "1.0.2",
     "@types/lodash": "4.14.149",
     "@types/node": "12.12.17",
     "@types/react": "16.9.16",
     "@types/react-dom": "16.9.4",
     "@types/react-leaflet": "2.5.0",
     "@types/react-redux": "7.1.7",
     "@types/react-router-dom": "5.1.3",
+    "@types/react-test-renderer": "16.9.2",
     "@types/redux-logger": "3.0.7",
+    "@types/redux-mock-store": "1.0.2",
     "axios-mock-adapter": "1.17.0",
+    "enzyme": "3.11.0",
+    "enzyme-adapter-react-16": "1.15.2",
     "eslint-config-prettier": "6.7.0",
     "eslint-plugin-prettier": "3.1.1",
+    "husky": "4.2.3",
+    "lint-staged": "10.0.8",
```

Comment:
Please review: 
- [husky](https://www.npmjs.com/package/husky) -> allows to add Git hooks without fudging with the .git folder
-  [lint-staged](https://www.npmjs.com/package/lint-staged) -> Run linters against staged git files (before you commit them)

More on why I added them below...